components/mpas-seaice/cime_config/buildnml

```diff
@@ -213,6 +213,12 @@ def buildnml(case, caseroot, compname):
         grid_prefix += 'mpassi.EC30to60E2r2-1900.rstFromG-anvil'
         decomp_date += '200908'
         decomp_prefix += 'mpas-seaice.graph.info.'
+    elif ice_grid == 'ECwISC30to60E2r1':
```

Comment:
@jonbob my thought was to only point to the one-month G-case restarts as ICs for this grid (and effectively not support cold-starts). Since you're currently reworking the 'ICG' tag functionality, maybe it's best if I wait for that to be merged and make this grid consistent with that?

Reply:
That sounds good, @darincomeau. I think that makes the most sense, unless you need this in master sooner. In that case, you're probably the only person who's going to run with this configuration and know about the limitations for which compsets to use.

Reply:
@darincomeau was pushing on this at my request eventually, I want to test the new EOS in https://github.com/MPAS-Dev/MPAS-Model/pull/313 with this mesh but it's definitely not urgent.